Superior Tiling

taibeled opened the pull request:
@Hawkguye, would you be okay with merging your changes into master? This would mean that your API key would have more requests.

Hawkguye commented:
It's totally fine, but my free api key has a 150,000 tile request limit, and 8% of it has been used only in the last 3 days. I don't think my api key would handle your traffic throughout the month. I will be glad to help though, perhaps we can open up more free accounts but it will just be a temporary solution.

taibeled commented:
Yes, that is the difficult part. So much traffic went to OpenRailwayMap—the free tool for railway drawing that was toggled off by default—that this website is now banned from accessing it.

Hawkguye commented:
Yes, I checked the leaflet provider demo, there aren't any more options other than this Thunderforest one and OpenRailwayMap, which is really unfortunate. Worst case would be to upgrade the Thunderforest account to the next tier, which has 10x the quota but is $125 a month. Thank you for your devotion into the project, really love it!

taibeled commented:
I added this (5e5543c384cc5758ca15dd54b1095006163f411c), but with a toggle, and it not being shown all the way zoomed out. Hopefully that should cut down on some API usage. I also added a fallback in case the API overloads. Thanks!

Hawkguye commented:
Hey, just to let you know that the quota has already been exceeded (161%). So the map tiles will probably stop working shortly.

taibeled commented:
> Hey, just to let you know that the quota has already been exceeded (161%). So the map tiles will probably stop working shortly.

Somehow, the map tiles still seem to be working. Thanks for the update!

taibeled commented:
@Hawkguye do you know what percent the quota has reached now?

Hawkguye commented:
245%. Thunderforest reached out to me asking for upgrade. They were being nice not to cut it immediately, they will soon cut it if I don't upgrade.
